### DISCUSSION TITLE
New analysis report ready for BladeBit CUDA performance from Science of Mining

### DISCUSSION THREAD

**Discussion by scienceofmining** [Chia-Network/bladebit#379] [Show and tell]
Hello BladeBit CUDA fans, we have produced a new performance report on the performance and power efficiency of BladeBit CUDA. The report is available at our tech blog: https://scienceofmining.com/20230828-BladeBit/BladeBit%20CUDA%20Performance%20Analysis-1.html 

Feel free to check it out and let us know your feedback. We surveyed the Xeon W5-2455X, Core i9 12900k, Core i9 11900k, and Core i9 9900k for farming efficiency and also the RTX 3070 + RTX 3090. 

Sept 2, 2023 update: We released a how-to for getting started with BladeBit CUDA benchmarking: https://scienceofmining.com/bb_guide/guide.html

-- The Science of Mining team

**Comment by hajes** [comment]
you can achieve lower GPU wattage by forcing lower core/mem clock

My old Quadro M4400 used 50W (267 TiB C5 plots)

underclocked 18W

Lookup times same. bladebit simulation is weird - my GPU supposed to not even farm such a big farm...yet it easily does ~0.5s lookups. In case of CPU, simulation may work. AMD Ryzen 5 4500 had lookup times ~10s with same farm size

modern RTX GPUs may be even more efficient than old Quadro GPUs.

**Comment by scienceofmining** [comment]
Interesting idea to reduce the clocks on the GPU to reduce consumption, makes sense since there's still ample headroom to meet the response times. A deeper dive into this area could be useful.

**Comment by hajes** [comment]
GPU is efficient but adds too much overhead. On small farms <300TiB problem.

So far my GPU can keep up at lowest clock.

No idea what will happen once I replot all, and farm size hits 1+ PiB

So far `bladebit_cuda simulate` shows weird results. I shouldn't be able to even farm what I have now for testing purposes.

**Reply by scienceofmining** [reply]
It sounds like your conclusions mirror some of the conclusions of our article. It would be interesting to compare the results of the simulation vs an actual farm to better validate the simulator report.

**Comment by hajes** [comment]
There is an issue with setting core, mem clock on consumer GPUs. I can set core clock, and force power limit only on `RTX 3060 Ti`. Underclocking is fully supported only on workstation/server GPUs Quadro, Tesla.

**Reply by scienceofmining** [reply]
Interesting. I used to underclock my core clocks on the original RTX 3070/3080/3090 before on Windows and I think on Linux too, but it's been a while. I'll do some checking as we get to that part of the testing to see what's the current situation. Thanks for the info.
